### DISCUSSION TITLE
Server locations

### DISCUSSION THREAD

**Discussion by R-Mull** [supabase/supabase#19818] [Questions] [answered]
As a newbie with zero Supabase experience, can anyone advise what server locations are available? 
I'm building my frontend on WeWeb but need to have the server based in Australia if possible. Cheers.

**Comment by GaryAustin1** [comment, marked as answer]
![image](https://github.com/supabase/supabase/assets/54564956/c2cb273b-7d16-41df-9544-8d3ba6268f33)

**Reply by R-Mull** [reply]
Thanks Gary. I couldn't find anything in the documentation. I guess I should have just played around a bit.
